### PR TITLE
Update auth-sds user guide

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Please provide a description for what this PR is for.
Update caveats section in https://preliminary.istio.io/docs/tasks/security/auth-sds/#caveats, as we have enabled SDS for control plane, and it is alpha feature.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[x] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/auth-sds/index.md [hunk 1]
```diff
@@ -354,8 +354,7 @@ To enable the pod security policy, perform the following steps:
 
 Currently, the SDS identity provision flow has the following caveats:
 
-* You still need secret volume mount for enabling the control plane security.
-  Enabling SDS for the control plane security remains a work in progress.
+* SDS for the control plane security is enabled by default, and it is alpha version.
```
Comment (myidpt) [inline content/en/docs/tasks/security/auth-sds/index.md]:
```suggestion
* The SDS feature is currently in Alpha phase.
```

Comment (myidpt) [reply content/en/docs/tasks/security/auth-sds/index.md]: I don't think we need to mention too much about the control plane security. Our users should naturally think the control plane security is using SDS.

File: content/en/docs/tasks/security/auth-sds/index.md [hunk 2]
```diff
@@ -354,8 +354,7 @@ To enable the pod security policy, perform the following steps:
 
 Currently, the SDS identity provision flow has the following caveats:
 
-* You still need secret volume mount for enabling the control plane security.
-  Enabling SDS for the control plane security remains a work in progress.
+* The SDS feature is currently in Alpha phase.
```
Comment (rcaballeromx) [inline content/en/docs/tasks/security/auth-sds/index.md]: Add link to the feature status page:
```suggestion
* SDS support is currently in [Alpha](/about/feature-stages/#security-and-policy-enforcement).
```

**Pull request merged**